changes (#177)
LIT-Protocol/docs: docs/intro/usecases.md

# Use Cases

> Developers can use the Lit Network to build applications that leverage decentralized and programmable keys at their core. Possible use cases include:

Reviewer: ```suggestion
Developers can leverage the Lit Network to build  decentralized applications with programmable keys at their core. Possible use cases include:
```
